v2/tools/generator/internal/astmodel/type_visitor_test.go

```diff
@@ -144,6 +144,12 @@ func TestIdentityVisitorReturnsEqualResult(t *testing.T) {
 		WithFunction(transmogrify).
 		WithFunction(skew)
 
+	enum := NewEnumType(StringType)
```

Comment:
Suggest renaming this to `stringEnum` to allow other tests to be written with `NewEnumType(IntType)` in the future.